pyclowder/client.py

```diff
@@ -168,7 +168,7 @@ def delete(self, path, params=None, headers=None):
             auth = None
         while True:
             try:
-                response = requests.delete(url, headers=headers, params=params,
+                response = requests.delete(url, params=params,
```

Comment:
problem is that this will also ignore any headers the user sets up, for example `X-Api-Key`. I think better is to not initialize it with `application/json` a few lines above.